src/common/lib/client/baserealtime.ts

```diff
@@ -24,6 +24,9 @@ class BaseRealtime extends BaseClient {
   _channels: any;
   connection: Connection;
 
+  // Make EventEmitter usable in other SDKs without changing public API or typings.
```

Comment:
I think this does better to explain why this line is here for other developers
```suggestion
  // private API to make EventEmitter usable in other SDKs
```

Reply:
I would say semi-private :) Probably word "internal" fits better, wdyt?

Reply:
yeah good shout, agreed :)

Reply:
Thanks both! Done.

Reply:
I don't have access to merge. Could someone merge for me please? 🙏